sdk/identity/identity/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- `AzureCliCredential`: Added support for the new response field which represents token expiration timestamp as time zone agnostic value. ([#28333](https://github.com/Azure/azure-sdk-for-js/pull/28333))
```

Comment:
@KarishmaGhiya we probably want to do a minor version bump here instead of a patch release right? And have a beta version release as well?

Reply:
Minor version bump with a beta first.

Reply:
4.1.0-beta.1

Reply:
Done 👍